test/retiarii_test/lightning/test_darts.py

```diff
@@ -0,0 +1,107 @@
+from pytorch_lightning.trainer.trainer import Trainer
+from torch.utils.data import dataloader
+from nni.retiarii.nn.pytorch.component import Repeat
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from nni.retiarii.evaluator.pytorch.lightning import DataLoader
+from torchvision import transforms
+from torchvision.datasets import MNIST
+from nni.retiarii.nn.pytorch import LayerChoice
+import pytorch_lightning as pl
+from nni.retiarii.oneshot.pytorch.utils import ParallelTrainValDataLoader
+
+class DepthwiseSeparableConv(nn.Module):
+    def __init__(self, in_ch, out_ch):
+        super().__init__()
+        self.depthwise = nn.Conv2d(in_ch, in_ch, kernel_size=3, groups=in_ch)
+        self.pointwise = nn.Conv2d(in_ch, out_ch, kernel_size=1)
+
+    def forward(self, x):
+        return self.pointwise(self.depthwise(x))
+
+class Net(pl.LightningModule):
```

Comment:
The three tests have the same `Net`. You can have a `test_oneshot.py` and put three methods `test_darts()`, `test_enas()`, `test_proxyless()` into it.